tutorials/getting_started.md

```diff
@@ -0,0 +1,116 @@
+
+You can also directly load an image into the viewer from the command line by passing the path to the image as an argument as follows
+```sh
+napari my_image.png
+```
+If the image is multichannel (i.e. `RGB` or `RGBA`) use the `-m` or `--multichannel` flag.
+
+![image](resources/launch_cli_image.gif)
+
+Launching **napari** directly from the command line is the simplest and fastest way to open the viewer, but it doesn't allow you to preprocess your images before opening them. It is also currently not possible to save images or other layer types directly from the viewer, but we'll be adding support for this functionality soon as discussed in #379.
+
+### python script usage
+
+To launch **napari** from a python script, inside your script you should import `napari`, instantiate a qt gui, and create a `Viewer` object.
+
+Inside your script you should include:
+
+```python
+import napari
+
+# instantiate qt gui
+with napari.gui_qt():
+    # create an empty viewer object
+    viewer = napari.Viewer()
+
+    # custom code to add data here
+    viewer.add_image(my_image_data)
+```
+
+then run your script from the command line to launch the viewer with your data:
+```sh
+python my_example_script.py
+```
+
+See the scripts inside the [`examples`](../examples/add_points.py) in the main repository for examples of using napari this way.
+
+![image](resources/launch_script.gif)
+
+The advantage of launching **napari** from a python script is that you can preprocess your images and add multiple layers before displaying the viewer. It is possible to save out any edits made inside the viewer by including saving code outside the `with napari.gui_qt():` context, however this can require knowing what you want to save when you write the script. You can load additional images into the viewer using the `File/Open` dropdown menu or dragging and dropping them directly on the viewer, similar to if you had launched **napari** from the command line.
+
+### jupyter notebook / ipython usage
+
+To launch **napari** from a jupyter notebook or ipython console, first instantiate a qt gui and then import `napari` and create a `Viewer` object.
+
+```python
+# instantiate qt gui
+%gui qt5
+
+import napari
+
+# create an empty viewer object
+viewer = napari.Viewer()
+
+# custom code to add data here
+viewer.add_image(my_image_data)
+```
+Note that the `Qt` gui can take a few seconds to be created and if you create the `Viewer` before it is finished the kernel will die and the viewer will not launch.
```

Comment:
before it is finished, the kernel will die

(add comma)

Reply:
fixed